airflow/hooks/S3_hook.py

```diff
@@ -318,7 +318,8 @@ def load_file(self,
                   key,
                   bucket_name=None,
                   replace=False,
-                  encrypt=False):
+                  encrypt=False,
+                  ACL=None):
```

Comment:
lowercase name will be still better even if it is an acronym

Reply:
@potiuk Yes, I will replace `ACL` to `acl`.